couch2pg/tests/unit/setup.spec.js

```diff
@@ -13,6 +13,10 @@ describe('setup', () => {
       end: sinon.stub(),
     };
 
+    pgClient.query.withArgs(`SELECT 1 FROM v1.whatever LIMIT 1;`).rejects({
```

Comment:
Please add a unit test where the table already exists and doesn't get created on startup.